todo_app/app.py

```diff
@@ -1,9 +1,9 @@
 from flask import Flask, render_template, redirect
 from flask.globals import request
-from todo_app.data.trello_items import get_item, add_item, get_items, save_item
```

Comment:
These todo_app references weren't working in the container for some reason. Removing them seemed to fix the issue